hphp/hack/src/errors/errors.ml

```diff
@@ -13,46 +13,31 @@ open String_utils
 
 type error_code = int [@@deriving eq, ord, show]
 
+type 'a message = 'a * string [@@deriving eq, ord, show]
 (** We use `Pos.t message` and `Pos_or_decl.t message` on the server
     and convert to `Pos.absolute message` before sending it to the client *)
-type 'a message = 'a * string [@@deriving eq, ord, show]
 
 let get_message_pos (msg : 'a message) = fst msg
 
 let get_message_str (msg : 'a message) = snd msg
 
-type phase =
-  | Init
-  | Parsing
-  | Naming
-  | Decl
-  | Typing
+type phase = Init | Parsing | Naming | Decl | Typing
```

Comment:
Looks like `ocamlformat` isn't picking up the configuration file, which is here:

https://github.com/facebook/hhvm/blob/master/hphp/hack/.ocamlformat

Try moving the `.ocamlformat` file to the root of the git repository? Hopefully that will fix it. If it does, you can create a pull request for that too :)

Reply:
I moved the `.ocamlformat` file with `~/hhvm$ mv hphp/hack/.ocamlformat .ocamlformat` and then in `errors.ml` I pressed `option+Shift+F` to reformat but nothing changed. Could you replicate this new formatting on your machine?